sources/packages/web/src/composables/useRules.ts

```diff
@@ -38,18 +38,70 @@ export function useRules() {
     return `${fieldName} is required.`;
   };
 
+  /**
+   * Check if a value is truthy.
+   * All values are truthy except false, 0, -0, 0n,
+   * "", null, undefined, and NaN.
+   * @param value value to be checked.
+   * @param fieldName friendly field name to be added
+   * to the validation message.
+   * @returns true if the value is truthy,
+   * otherwise a validation message.
+   */
+  const truthyRule = (value: string, fieldName: string) => {
+    if (value) {
+      return true;
+    }
+    return `${fieldName} is required.`;
+  };
+
   const checkOnlyDigitsRule = (value: string, fieldName: string) => {
     if (!/^\d+$/.test(value)) {
       return `${fieldName} is invalid.`;
     }
     return true;
   };
 
+  /**
+   * Checks if a number is between a range (inclusive check).
+   * @param value value to be checked.
+   * @param min min value allowed (inclusive check).
+   * @param max max value allowed (inclusive check).
+   * @param fieldName optional field name to be added to the message.
+   * @param numberFormatter optional formatter to display min and max
+   * on the validation message.
+   * @returns true if the value is between the min and max value,
+   * otherwise a validation message.
+   */
+  const numberRangeRule = (
+    value: string,
+    min: number,
+    max: number,
+    fieldName: string,
+    numberFormatter: (value: number | string) => string,
+  ) => {
+    const numberValue = parseFloat(value);
+    if (numberValue < min || numberValue > max) {
+      let minText = min.toString();
+      let maxText = max.toString();
+      if (numberFormatter) {
+        minText = numberFormatter(minText);
+        maxText = numberFormatter(maxText);
+      }
+      return `${
+        fieldName ?? "Value"
```

Comment:
Why are we performing null check on fieldname? I see that it is required in the method.

Reply:
My intention was to leave it optional but I forgot that the same message goes to the top of the form, so not including the field name would be confusing to the user. I will be making it required.